### ISSUE TITLE
reshape contiguous array

### ISSUE THREAD

**Issue by promitmoitra** [mortacious/numba-kdtree#4]
On line 64 of numba_kdtree/kdtree.py
Instead of
        return np.ascontiguousarray(x_tmp.reshape(-1, n_features))
did you mean
        return np.ascontiguousarray(x_tmp).reshape(-1, n_features)
because I was running a fairly simple piece of code, and got an error saying reshape is only supported for contiguous arrays.

**Comment by mortacious** [comment]
It works both ways so your problem should be fixed now.

**Comment by promitmoitra** [comment]
So I am unclear about it working both ways, because:
return np.ascontiguousarray(x_tmp.reshape(-1, n_features))
implies first reshape x_tmp, then convert it to a contiguous array. The error pointed to this specific issue, that reshape is not supported for non-contiguous arrays.
Whereas, return np.ascontiguousarray(x_tmp).reshape(-1, n_features) implies first convert x_tmp to a contiguous array, and then reshape it. My problem was fixed after I made this change in the source locally.
In any case, thank you for the numba compatible KDTree implementation.
